app/jobs/scheduled/automatic_translation_backfill.rb

```diff
@@ -24,10 +24,10 @@ def fetch_untranslated_model_ids(model, content_column, limit, target_locale)
             SELECT m.id
             FROM #{model.table_name} m
             #{limit_to_public_clause(model)}
-            WHERE m.deleted_at IS NULL
-              AND m.#{content_column} != ''
-              AND m.user_id > 0
-              #{max_age_clause}
+            WHERE m.#{content_column} != ''
+              #{not_deleted_clause(model)}
+              #{non_bot_clause(model)}
+              #{max_age_clause(model)}
```

Comment:
I'm wondering if it makes sense to just make this `backfill_clause(model)` or something instead of having the logic of `model == Post || model == Topic` spread over 3 methods.